src/l500/l535-color.cpp

```diff
@@ -0,0 +1,33 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2018 Intel Corporation. All Rights Reserved.
+
+#include "l500-color.h"
```

Comment:
Remove this include

Reply:
its needed (line 14)

Reply:
But we're deriving from l500_color... it should be covered by the header...

Reply:
Done